scripts/zones/VeLugannon_Palace/IDs.lua

```diff
@@ -31,6 +33,7 @@ zones[tpz.zone.VELUGANNON_PALACE] =
     },
     npc =
     {
+        QM3             = 17502583,
```

Comment:
any chance we could use better naming convention then "QM3"? I know there are some client limitations here so maybe ignore this. Teo can probably weigh in.

Reply:
Agreed.  One of my eventual TODOs is getting rid of all these numbered QMs in npc_list/scripts, and replacing them with more meaningful names.  Client NPC ID shifts sometimes add/remove/shift QMs in a way that messes up the order within a zone.

Reply:
Just need to know what name to change it to.

Reply:
I would say just leave it for now, unfortunately. That way it still matches up with the name in the database.